content/en/blog/_posts/2022-08-04-cgroupv2-ga.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: blog
+title: "CGroup V2 Goes GA in 1.25"
+date: 2022-08-04
+slug: cgroupv2-ga-1-25
+---
+Authors: David Porter (Google), Mrunal Patel (RedHat)
+
+Kubernetes 1.25 brings cgroup v2 to GA, enabling kubelet to use the latest resource management capabilities.
+
+## What are cgroups?
+
+One of the most important aspects of Kubernetes is [resource management](/docs/concepts/configuration/manage-resources-containers/) which is concerned with how to manage finite resources in clusters across nodes such as CPU, Memory, Disk, etc.
+
+Anytime you're  making use of resource management capabilities in Kubernetes like setting [Requests and Limits across CPU and Memory](/docs/concepts/configuration/manage-resources-containers/#requests-and-limits) for your pods, cgroups are being used.
+
+cgroups are a low level Linux kernel capability that underpin resource management functionality like limiting CPU usage or setting memory limits.
+
+There are two versions of cgroups offered in the Linux kernel cgroups v1 and cgroups v2.
+
+## What is cgroup v2?
+
+cgroup v2 is the next version of the Linux cgroup API. cgroup v2 provides a unified control system with enhanced resource management capabilities.
+
+cgroup v2 has been development in the Linux Kernel since 2016 and in recent years has matured across the container ecosystem. With Kubernetes 1.25, cgroup v2 support is graduating to GA.
+
+Many new Linux distributions have switched over to cgroup v2 by default so it's important that Kubernetes continues to work well on these new updated distros.
```

Comment:
```suggestion
cgroup v2 has been in development since 2016, and has matured in the container ecosystem. Many Linux distributions have switched to use cgroup v2 by default. It's important that Kubernetes continues to work well
on these updated distributions. In Kubernetes 1.25, cgroup v2 support is graduating to GA.
```

Is GA what we use or stable?

Reply:
Looking through older blogs, look like stable, let me rename GA to stable throughout.